src/steadystateproblem.cpp

```diff
@@ -527,10 +586,14 @@ void SteadystateProblem::getSteadystateSimulation(Solver *solver,
          only direction w.r.t. current t
          */
         solver->step(std::max(t, 1.0) * 10);
-        solver->writeSolution(&t, x, dx, sx);
+        if (backward) {
+            solver->writeSolution(&t, xB, dx, sx, xQ);
+        } else {
+            solver->writeSolution(&t, x, dx, sx, xQ);
+        }
 
         /* Check for convergence */
-        converged = checkConvergence(solver, model);
+        converged = checkConvergence(solver, model, sensitivityFlag);
```

Comment:
I was thinking whether it might make sense to do this check for convergence only once in 10 steps, because I have the feeling that it may be time-consuming if done always...

Reply:
Okay. I can at some point check this for the human benchmark model. Not a big deal to change 3 lines, recompile and monitor cpu time